ibm/mas_devops/roles/suite_app_config/vars/health.yml

```diff
@@ -13,8 +13,6 @@ mas_app_settings_demodata: "{{ lookup('env', 'MAS_APP_SETTINGS_DEMODATA') | defa
 mas_app_settings_tablespace: "{{ lookup('env', 'MAS_APP_SETTINGS_TABLESPACE') | default('MAXDATA', true)}}"
 mas_app_settings_indexspace: "{{ lookup('env', 'MAS_APP_SETTINGS_INDEXSPACE') | default('MAXINDEX', true)}}"
 
-server_bundles_jms_add_server_config_secret_name: "{{ mas_workspace_id }}-manage-jms-additional-server-config"
-server_bundles_add_server_config_secret_name: "{{ mas_workspace_id }}-manage-additional-server-config"
```

Comment:
Would it be easier to set these as the new values rather than remove them and have the new values in mulitple places? i.e. why not change ```server_bundles_add_server_config_secret_name: "{{ mas_workspace_id }}-manage-additional-server-config"``` to be ```server_bundles_add_server_config_secret_name: "{{ mas_workspace_id }}-{{ mas_app_id }}-d--sb0--asc--sn``` ?

Reply:
It would also make it easier to know what those are as `-d--sb0--asc--sn` means nothing to me.